lib/shopify/customer.ex

```diff
@@ -52,6 +78,20 @@ defmodule Shopify.Customer do
     }
   end
 
+  @doc """
+  Delete a single customer metafield
+
+  ## Examples
+    iex> Shopify.Customer.delete_customer_metafield("12345", "67890") |> Shopify.request(session)
+  """
+  @spec delete_customer_metafield(binary, binary) :: Shopify.Operation.t()
```

Comment:
Since this is on the `Customer` module adding `customer` to the function name is redundant. This can just be `delete_metafield/2`.